bitnami/kube-prometheus/templates/alertmanager/tls-secrets.yaml

```
@@ -0,0 +1,32 @@
+{{- if and .Values.alertmanager.enabled .Values.alertmanager.ingress.enabled }}
+{{- if .Values.alertmanager.ingress.secrets }}
+{{- range .Values.alertmanager.ingress.secrets }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .name }}
+  namespace: {{ $.Release.Namespace }}
+  labels: {{- include "kube-prometheus.alertmanager.labels" . | nindent 4 }}
+    type: kubernetes.io/tls
+    data:
+      tls.crt: {{ .certificate | b64enc }}
+      tls.key: {{ .key | b64enc }}
+---
+{{- end }}
+{{- end }}
+{{- if and .Values.alertmanager.ingress.tls (not .Values.alertmanager.ingress.certManager) }}
+{{- $ca := genCA "alertmanager-ca" 365 }}
+{{- $cert := genSignedCert .Values.alertmanager.ingress.hostname nil (list .Values.alertmanager.ingress.hostname) 365 $ca }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ printf "%s-tls" .Values.alertmanager.ingress.hostname }}
+  namespace: {{ .Release.Namespace }}
+  labels: {{- include "kube-alertmanager.prometheus.labels" . | nindent 4 }}
```

This template is wrong:
```suggestion
  labels: {{- include "kube-prometheus.alertmanager.labels" . | nindent 4 }}
```

---

Done